sentinelone/plugin.spec.yaml

```
@@ -3,7 +3,7 @@ extension: plugin
 products: [insightconnect]
 name: sentinelone
 title: SentinelOne
-version: 1.2.1
+version: 1.4.0
```

```suggestion
version: 1.3.0
```
Doesn't appear 1.3.0 has been released yet?

---

@zyoutz-r7 
It is correlated with blacklist action. There is 1.3.0 version so here we set it to 1.4.0 to not conflict each other.

---

We just merged the 1.3.0 PR. This is 1.4.0 now 👍